dark/proteins.py

```diff
@@ -302,11 +305,13 @@ def addFile(self, filename, fp):
                     'Protein %r already seen for pathogen %r sample %r.' %
                     (proteinName, pathogenName, sampleName))
 
+            readsFilename = join(outDir, '%d.%s' % (index, self._format))
```

Comment:
Because I re-use the filename in the subsequent code (if read lengths are being saved).